sources/packages/api/src/services/institution-location/institution-location.service.ts

```diff
@@ -56,6 +57,57 @@ export class InstitutionLocationService extends RecordDataModelService<Instituti
     return this.repo.save(saveLocation);
   }
 
+  /**
+   * Updating institution location.
+   * @param data Payload of updated data.
+   * @param locationId Location Id to update.
+   * @returns Updated location as InstitutionLocation.
+   */
+  async updateLocationPrimaryContact(
+    data: Partial<InstitutionLocationPrimaryContactModel>,
```

Comment:
why is this `Partial`? I am seeing that the payload controller is passing and InstitutionLocationPrimaryContactModel have same properties